test/sequential/test-timers-block-eventloop.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+const t1 = setInterval(() => {
+  common.busyLoop(12);
+}, 10);
+
+const t2 = setInterval(() => {
+  common.busyLoop(15);
+}, 10);
+
+const t3 = setTimeout(() => {
+  throw new Error('eventloop blocked!');
+}, 100).unref();
+
+setTimeout(function() {
+  fs.stat('./nonexistent.txt', (err, stats) => {
+    clearInterval(t1);
+    clearInterval(t2);
+    clearInterval(t3);
+  });
```

Comment:
This is somewhat better but I still do not understand why there is any need for a extra function call. It should be sufficient without the `fs.stat` call. Just the plain set timeout is necessary as far as I can tell.

Reply:
Just make sure a different event is blocked. Per my local test, timer with different interval is not blocked.